controllers/nodefeaturediscovery_controls.go

```diff
@@ -198,23 +196,22 @@ func ClusterRoleBinding(n NFD) (ResourceStatus, error) {
 
 	// found states if the ClusterRoleBinding was found
 	found := &rbacv1.ClusterRoleBinding{}
-	logger := log.WithValues("ClusterRoleBinding", obj.Name, "Namespace", obj.Namespace)
 
 	// It is also assumed that our ClusterRoleBinding has a defined
 	// Namespace
 	obj.Subjects[0].Namespace = n.ins.GetNamespace()
 
-	logger.Info("Looking for")
+	klog.Info("Looking for ClusterRoleBinding %q on Namespace %q", obj.Name, obj.Namespace)
```

Comment:
supert nit: in/on namespace, use consistently 😄 

```suggestion
	klog.Info("Looking for ClusterRoleBinding %q in Namespace %q", obj.Name, obj.Namespace)
```